include/seqan/graph_types/graph_impl_tree.h

```diff
@@ -639,6 +639,57 @@ getAdjacencyMatrix(Graph<Tree<TCargo, TSpec> > const& g,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TVector, typename TCargo, typename TSpec, typename TVertex>
+inline void
+getVertexAdjacencyVector(TVector & vectIn,
+                         TVector & vectOut,
+                         Graph<Tree<TCargo, TSpec> > const & g,
+                         TVertex const & vertex)
+{
```

Comment:
Made the required modifications, a single implementation is located in the graph_impl_direct.h file with the name _getVertexAdjacencyVector, that function is then called from graph tree and graph directed type